reconcile/utils/oc.py

```diff
@@ -1038,6 +1040,18 @@ def __init__(self, clusters=None, namespaces=None,
         else:
             raise KeyError('expected one of clusters or namespaces.')
 
+    def set_jh_ports(self, jh):
+        # This will be replaced with getting the data from app-interface in
+        # a future PR.
+        jh['remotePort'] = 8888
+        key = f"{jh['hostname']}-{jh['remotePort']}"
```

Comment:
Why `-` instead of `:` in the key? It is a perfectly valid component of a FQDN and can get confusing...